packages/atoms/src/Hint.tsx

```diff
@@ -26,7 +25,7 @@ const hintTextStyle = prepareNativeStyle<{ color: Color }>((utils, { color }) =>
     marginLeft: 6,
 }));
 
-const hintVariants: Record<HintVariant, { iconType: IconType; color: Color }> = {
+const hintVariants: Record<HintVariant, { iconType: IconVariant; color: Color }> = {
```

Comment:
`variant` is definitely not good name because it's suggesting that there are some variants of one same thing :D But I agree that `iconType` is also not good name.

I did some research on other libraries and they are using just plain `name` in Icon component and when passed to other components like Hint use `iconName`. That seems to as best option and it's also same as [react-native-vector-icons](https://github.com/oblador/react-native-vector-icons#icon-component) which is nice.